fido/fido.py

```diff
@@ -350,11 +364,12 @@ def get_extension(self, format):
         return format.find('extension').text
 
     def print_matches(self, fullname, matches, delta_t, matchtype=''):
-        """The default match handler.  Prints out information for each match in the list.
-           @param fullname is name of the file being matched
-           @param matches is a list of (format, signature)
-           @param delta_t is the time taken for the match.
-           @param matchtype is the type of match (signature, containersignature, extension, fail)
+        """
+        The default match handler. Prints out information for each match in the list.
```

Comment:
Nitpick: If a docstring is multiline, the first line should be a summary, followed by a blank line, then the rest of it.  See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings